json/tbd_warehouse5.json

```diff
@@ -0,0 +1,41 @@
+{
+  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "description": "test_warehouse.osm with spacetype PSI values",
+  "psis": [
+    {
+      "id": "Warehouse Office",
+      "rimjoist": 0.300,
+      "parapet": 0.300,
+      "fenestration": 0.300,
+      "corner": 0.300,
+      "balcony": 0.300,
+      "party": 0.300,
+      "grade": 0.300
+    },
+    {
+      "id": "Warehouse Fine",
+      "rimjoist": 0.500,
+      "parapet": 0.500,
+      "fenestration": 0.500,
+      "corner": 0.500,
+      "balcony": 0.500,
+      "party": 0.500,
+      "grade": 0.500
+    }
+  ],
+  "building": [
+    {
+      "psi": "regular (BETBG)"
+    }
+  ],
+  "spacetypes": [
+    {
+      "id": "Warehouse Office",
+      "psi": "Warehouse Office"
+    },
+    {
+      "id": "Warehouse Fine",
+      "psi": "Warehouse Fine"
+    }
+  ]
```

Comment:
Default space-type-specific PSI sets, similar to OpenStudio default space-type construction sets.